exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/OracleDialect.kt

```diff
@@ -318,7 +318,7 @@ open class OracleDialect : VendorDialect(dialectName, OracleDataTypeProvider, Or
 
     override fun createDatabase(name: String): String = "CREATE DATABASE ${name.inProperCase()}"
 
-    override fun dropDatabase(name: String): String = "DROP DATABASE ${name.inProperCase()}"
+    override fun dropDatabase(name: String): String = "DROP DATABASE"
```

Comment:
Could you tell me why don't we need the name in drop statement?

Reply:
@e5l [Documentation](https://docs.oracle.com/en/database/oracle/oracle-database/18/sqlrf/DROP-DATABASE.html#GUID-4FFC1AF5-538D-4882-8979-7A9957492A23) shows that syntax should not have the database name in the drop statement.

This can be confirmed in the test if `SchemaUtils.dropDatabase(dbName)` is pulled out of the try block (or if the `cause.cause` is logged in the catch block) as it fails with this exception:
```
java.sql.SQLSyntaxErrorException: ORA-00933: SQL command not properly ended.
Statement(s): DROP DATABASE JETBRAINS
```

Removing the name resolves this exception (but then leads to the next exception - `ORA-65040: operation not allowed from within a pluggable database`)